TLM/TLM/State/GlobalConfig.cs

```diff
@@ -14,7 +14,7 @@ namespace TrafficManager.State {
     public class GlobalConfig : GenericObservable<GlobalConfig> {
         public const string FILENAME = "TMPE_GlobalConfig.xml";
         public const string BACKUP_FILENAME = FILENAME + ".bak";
-        private static int LATEST_VERSION = 20;
+        private static int LATEST_VERSION = 22;
```

Comment:
You don't need to update config version if your changes are affecting only debug build of the mod.